### PR TITLE
docs: Add contents to CONTRIBUTING.md

### LINKED ISSUE #152: Improve contributing documentation to add install details
## Summary

Currently it is hard for a newcomer to start contributing because of some missing details (like how to run), that currently can only be accessed via the generic Jenkins contributing documentation at https://github.com/jenkinsci/.github/blob/master/CONTRIBUTING.md. Would like to prepare a doc PR to fix this issue to add to https://github.com/jenkinsci/conventional-commits-plugin/blob/main/.github/CONTRIBUTING.md for easy reference. Please assign me this issue if possible. 

## Reference

* https://github.com/jenkinsci/conventional-commits-plugin/blob/main/.github/CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by slim-patchy**
## Description
Fixes #152 

Added documentation to the CONTRIBUTING.md file for this repository in particular. Added details about how to **run** and **test**, as well as a gentle reminder about checking if Jenkins is installed before running this plugin. 

## Checklist
- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -12,21 +12,30 @@ Before you begin, check out our [Developer Guidelines](https://www.jenkins.io/do
...
 Go through our existing issues to find one that interests you. You could also use `labels` as filters, to find the best issue for you.
 
+### Run the plugin
+
+To run the plugin, use the following command, as described in the [official documentation](https://www.jenkins.io/doc/developer/tutorial/run/) on running a plugin:
+```shell
+mvn hpi:run
+```
+
+For this to be successful, please make sure Jenkins is already installed but not already running. The default URL where Jenkins is set up in the browser is [http://localhost:8080/jenkins/](http://localhost:8080/jenkins/), where Jenkins can be accessed once up and running.
```
Comment (philippart-s) [inline .github/CONTRIBUTING.md:30]: for me it's not mandatory to have a version of Jenkins installed to work.
I do not have a Jenkins installed (I use dicker image to launch a Jenkins instance) and the `mvn hpi:run`work.

Comment (slim-patchy) [reply .github/CONTRIBUTING.md:30]: How about if I change it up a bit and add something to the effect of "... please make sure to either have Jenkins installed, or a docker image of Jenkins is available for launch"?

Comment (philippart-s) [reply .github/CONTRIBUTING.md:30]: yes perfect !
I agree with you that it's better to have a real instance to do some real tests !

Comment (slim-patchy) [reply .github/CONTRIBUTING.md:30]: Sure, let me make the change now

**Review by philippart-s — APPROVED** [review]
:rocket: 
Thanks for the work !

**Pull request merged**